tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -177,6 +181,154 @@ TEST( OutputReportTabularTest, GetUnitConversion )
 
 }
 
+TEST( OutputReportTabularTest, AllocateLoadComponentArraysTest )
+{
+	ShowMessage( "Begin Test: OutputReportTabularTest, AllocateLoadComponentArraysTest" );
+
+	AllocateLoadComponentArraysDoAllocate = true;
```

Comment:
It is required when feeding IDF objects as strings, however, I recommend using it for nearly any unit test. It takes care of all resetting of global state and has many other helper functions to make checking assertions in unit tests easier.